Gopkg.toml

```diff
@@ -0,0 +1,42 @@
+# Gopkg.toml example
+#
+# Refer to https://golang.github.io/dep/docs/Gopkg.toml.html
+# for detailed Gopkg.toml documentation.
+#
+# required = ["github.com/user/thing/cmd/thing"]
+# ignored = ["github.com/user/project/pkgX", "bitbucket.org/user/project/pkgA/pkgY"]
+#
+# [[constraint]]
+#   name = "github.com/user/project"
+#   version = "1.0.0"
+#
+# [[constraint]]
+#   name = "github.com/user/project2"
+#   branch = "dev"
+#   source = "github.com/myfork/project2"
+#
+# [[override]]
+#   name = "github.com/x/y"
+#   version = "2.4.0"
+#
+# [prune]
+#   non-go = false
+#   go-tests = true
+#   unused-packages = true
+
+
+[[constraint]]
+  name = "github.com/fatih/color"
+  version = "1.7.0"
+
+[[constraint]]
+  branch = "master"
+  name = "github.com/golang-collections/collections"
+
+[[constraint]]
+  branch = "master"
+  name = "github.com/jesseduffield/gocui"
+
+[prune]
+  go-tests = true
+  unused-packages = true
```

Comment:
@jesseduffield Hi! Love the project.

I would recommend adding `non-go = true` as well to `Gopkg.toml` this will bring down size a bit. I do this is all my golang projects.
Ex:
```toml
[prune]
  go-tests = true
  unused-packages = true
  non-go = true
```

Brings down vendor dir a bit on this brnach when I added and re ran `dep ensure`.
Before:
```shell
$ du -ah lazygit/ | grep -w "lazygit//vendor$"
6.0M    lazygit//vendor
```
After:
```shell
$ du -ah lazygit/ | grep -w "lazygit//vendor$"
5.8M    lazygit//vendor
```